doc/source/user_guide/dsintro.rst

```diff
@@ -87,8 +87,8 @@ index will be pulled out.
 
 **From scalar value**
 
-If ``data`` is a scalar value, an index must be
-provided. The value will be repeated to match the length of **index**.
+If ``data`` is a scalar value, the value will be repeated to match
+the length of **index**.
```

Comment:
It seems to me we should also indicate what happens if no `index` is provided.

Reply:
Something like this?

If ``data`` is a scalar value, the value will be repeated to match
the length of **index**. If the **index** is not provided, it defaults to ``RangeIndex (0, 1, 2, …, n)``.